static/js/components/ErrorMessage.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import LoginButton from '../containers/LoginButton';
+import { Alert } from 'react-bootstrap';
+
+
+class ErrorMessage extends React.Component {
+  static propTypes = {
+    errorInfo:   React.PropTypes.object.isRequired,
+  };
+
+  state = {
+    alertVisible: true
+  };
+
+  render() {
+    if (!this.state.alertVisible) {
+      return null;
+    }
+
+    const {errorInfo} = this.props;
```

Comment:
You can use destructuring to pull out pieces of `errorInfo`:

    const { errorInfo: { errorStatusCode } } = this.props;

Will produce `errorStatusCode`, possibly with `undefined` if the key doesn't exist.